### PR TITLE
RDF regenerated based on last CDE templates

### PR DESCRIPTION
**Pull request by pabloalarconm**

### CODE REVIEW

File: example-data/turtle/sio-model/pseudonymNew.ttl [hunk 1]
```diff
@@ -1,13 +1,15 @@
 @prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
 @prefix obo: <http://purl.obolibrary.org/obo/> .
 @prefix sio: <http://semanticscience.org/resource/> .
+@prefix edam: <http://edamontology.org/> .
```
Comment (rajaram5) [inline example-data/turtle/sio-model/pseudonymNew.ttl:4]: can u remove this prefix?

File: example-data/turtle/sio-model/phenotyping.ttl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    rdfs:label "comparative phenotypic assessment"^^xsd:string;
+    sio:SIO_000229 :phenotypicoutput_ .
+
+
+:phenotypicoutput_ a sio:SIO_000015, obo:NCIT_C102741 ;
+    rdfs:label "Cerebellar ataxia"^^xsd:string ;
+    sio:SIO_000669 "1992-01-01"^^xsd:date;
+    sio:SIO_000628 :phenotypicattribute_ .
+
+:phenotypicattribute_ a SIO:000614, sio:SIO_0100565, <https://hpo.jax.org/app/browse/term/HP:0001251> .
```
Comment (rajaram5) [inline example-data/turtle/sio-model/phenotyping.ttl:33]: HPO url here is wrong? can you use this uri? obo:HP_0001251

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Minor comments

File: example-data/turtle/sio-model/pseudonymNew.ttl (hunk 1 above)
Comment (pabloalarconm) [reply example-data/turtle/sio-model/pseudonymNew.ttl]: Fixed

File: example-data/turtle/sio-model/phenotyping.ttl (hunk 2 above)
Comment (pabloalarconm) [reply example-data/turtle/sio-model/phenotyping.ttl]: Fixed

**Review by rajaram5 — APPROVED** [review]

**Pull request merged**